scripts/cbc_syslog_forwarder.py

```diff
@@ -0,0 +1,127 @@
+# *******************************************************
+# Copyright (c) VMware, Inc. 2020-2023. All Rights Reserved.
+# SPDX-License-Identifier: MIT
+# *******************************************************
+# *
+# * DISCLAIMER. THIS PROGRAM IS PROVIDED TO YOU "AS IS" WITHOUT
+# * WARRANTIES OR CONDITIONS OF ANY KIND, WHETHER ORAL OR WRITTEN,
+# * EXPRESS OR IMPLIED. THE AUTHOR SPECIFICALLY DISCLAIMS ANY IMPLIED
+# * WARRANTIES OR CONDITIONS OF MERCHANTABILITY, SATISFACTORY QUALITY,
+# * NON-INFRINGEMENT AND FITNESS FOR A PARTICULAR PURPOSE.
+
+"""CBC Syslog Forwarder main script"""
+
+import argparse
+import logging
+import logging.handlers
+import psutil
+import sys
+
+from cbc_syslog import poll
+from cbc_syslog.util import Config
+
+log = logging.getLogger(__name__)
+formatter = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+
+
+def main(args):
+    """Core CBC Syslog Forwarder Commands"""
+    config = Config(args.config_file)
+
+    if args.command == "poll":
+        succeeded = poll(config)
+
+    # elif args.command == "history":
```

Comment:
I assume these are just placeholders for future enhancements

Reply:
yep. I wanted to define them for future stories to implement